content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,130 @@ spec:
+    match:
+      any:
+      - resources:
+          kinds:
+          - Namespace
+    generate:
+      kind: NetworkPolicy
+      name: default-deny
+      namespace: "{{request.object.metadata.name}}"
+      synchronize: true
+      data:
+        metadata:
+          labels:
+            created-by: kyverno
+        spec:
+          podSelector: {}
+          policyTypes:
+          - Ingress
+          - Egress
+```
+
+By default, the above policy will not be applied when it is installed. This behavior can be configured via `generateExistingOnPolicyUpdate` attribute. If you set `generateExistingOnPolicyUpdate` to `true`, Kyverno will generate the target secret resource in existing namespaces on policy CREATE and UPDATE AdmissionReview events.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: generate-resources
+spec:
+  generateExistingOnPolicyUpdate: true
+  rules:
+  - name: generate-existing-networkpolicy
+    match:
+      any:
+      - resources:
+          kinds:
+          - Namespace
+    generate:
+      kind: NetworkPolicy
+...
+
+```
+
+### Generate PodDisruptionBudget for Existing Deployment
+
+This Cluster Policy will create a `PodDisruptionBudget` resource for existing deployments or new deployments.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: create-default-pdb
+spec:
+  generateExistingOnPolicyUpdate: true
+  rules:
+  - name: create-default-pdb
+    match:
+      resources:
+        kinds:
+        - Deployment
+    exclude:
+      resources:
+        namespaces:
+        - local-path-storage
+    generate:
+      apiVersion: policy/v1
+      kind: PodDisruptionBudget
+      name: "{{request.object.metadata.name}}-default-pdb"
+      namespace: "{{request.object.metadata.namespace}}"
+      synchronize: true
+      data:
+        spec:
+          minAvailable: 1
+          selector:
+            matchLabels:
+              "{{request.object.metadata.labels}}"
+
+```
+
+###
+To troubleshoot the policy application failure, you can inspect `UpdateRequest` Custom Resource  to get details. Successful `UpdateRequests` are automatically cleaned up by Kyverno.
+
+For example, if the corresponding permission is not granted to Kyverno, you should see this error in the `updaterequest.status`:
+
+```sh
+$ kubectl get ur -n kyverno
+NAME       POLICY               RULETYPE   RESOURCEKIND   RESOURCENAME           RESOURCENAMESPACE   STATUS   AGE
+ur-7gtbx   create-default-pdb   generate   Deployment     nginx-deployment       test                Failed   2s
+
+$ kubectl describe ur ur-7gtbx -n kyverno
+Name:         ur-7gtbx
+Namespace:    kyverno
+...
+
+status:
+  message: 'poddisruptionbudgets.policy is forbidden: User "system:serviceaccount:kyverno:kyverno-service-account"
+            cannot create resource "poddisruptionbudgets" in API group "policy" in the namespace "test"'
+ state: Failed
+```
+
+
+## Generating resources into existing namespaces using Selector
```

Comment:
Do we want to mark this as deprecated as your change introduced a general solution to generate on existing resources?